core/shared/src/main/scala/plotly/Trace.scala

```diff
@@ -1,5 +1,6 @@
 package plotly
 
+import scala.collection.immutable.Seq
```

Comment:
Same as above, let's keep the default `Seq`, even though it's slightly different in 2.12 and 2.13, here and in all `scala.collection.immutable.Seq` added below.